docs/getting_started.md

```diff
@@ -79,26 +79,26 @@ navigating to its metrics endpoint:
 
 ## Using the expression browser
 
-Let us try looking at some data that Prometheus has collected about itself. To
+Let's explore data that Prometheus has collected about itself. To
```

Comment:
Avoid contractions

Reply:
I was taught that as a kid, was erring on the side of being less awkward though.  Reverting.

Reply:
So was I, but apparently it's easier to read this way so that's what we use in our docs.